services/configurations.py

```diff
@@ -38,6 +38,6 @@
 
 def containsFlag(text):
```

Comment:
This is actually dead code, left over from flower. The `containsFlag` check is now done in the assembler instead.  
  
I've already removed it in my private fork for next ECSC, but forgot to do it here. You can remove this entire function.

Reply:
Removed, thanks!